compat/src/suspense-list.js

```diff
@@ -0,0 +1,72 @@
+import { Component, options } from 'preact';
+import { Suspense } from './suspense';
+
+// Hook for Suspense boundaries to ask for any extra work before rendering suspended children.
+options.__onSuspensionComplete = (vnode, cb) => {
```

Comment:
Hmm do we need this to be an option? Could we simplify and save some bytes and move the `_parent._component._modifySuspense` into Suspense directly?

Reply:
Sadly no.

A. That makes it really coupled, but I get it can be worthy of the savings.
B. Suspense isn't the only one who needs this logic. A `SuspenseList` can itself be a inside other other `SuspenseList`

e.g.
```jsx
<SuspenseList>
  <SuspenseList>...<SuspenseList>
  <Suspense>...<Suspense>
</SuspenseList>
```

Reply:
Whoa! Nested `SuspenseList` 🤯 Gotta think more about that